api/src/routes/topstats.js

```diff
@@ -49,10 +49,11 @@ module.exports = async (req, res) => {
     const [{ totalEdits }] = await db('users').sum('edit_count as totalEdits')
 
     const [{ numUsers }] = await db('users').count('id as numUsers')
-    const editsByCountry = await db('users')
-      .whereNotIn('osm_id', filteredUsers)
```

Comment:
Shouldn't we still check filteredUsers @kamicut ?

Reply:
Oh, that's a good point, I'll amend the sql